.github/workflows/ci.yaml

```diff
@@ -41,8 +41,7 @@ jobs:
       id: cache_timestamp
       shell: cmake -P {0}
       run: |
-        string(TIMESTAMP current_date "%Y-%m-%d-%H;%M;%S" UTC)
-        message("::set-output name=timestamp::${current_date}")
+        echo "timestamp=$(date +'%Y-%m-%dT%H:%M:%S')" >> $GITHUB_OUTPUT
```

Comment:
make this

`echo timestamp="$(date +'%Y-%m-%dT%H:%M:%S')"`

Reply:
Done.